client/blocks/taxonomy-manager/list-item.jsx

```diff
@@ -121,6 +121,7 @@ class TaxonomyManagerListItem extends Component {
 
 	render() {
 		const { canSetAsDefault, isDefault, onClick, term, translate, isJetpack } = this.props;
+		const name = this.getName();
```

Comment:
What do you think about dissolving `getName()` and inline it? All necessary props are destructured above